src/main/java/io/lindstrom/m3u8/parser/AbstractPlaylistParser.java

```diff
@@ -12,6 +13,7 @@
 
 public abstract class AbstractPlaylistParser<T extends Playlist, B> {
     private static final String EXTM3U = "#EXTM3U";
+    private static final String COMMENT = "#";
```

Comment:
Hmmm I just noticed I started to magic string out the "#" but then realised the code already had "#" usages and did not remove this...

Open to suggestions for either removing this or updating "#" usages to use the magic variable instead.

Reply:
I removed the static field.